### PR TITLE
Some QOI?

### PR DESCRIPTION
**Pull request by Dante-Broggi**
The commit titles are self-descriptive.

### CODE REVIEW

File: Sources/Pathos/pathAnalysis.swift [hunk 1]
```diff
@@ -64,7 +63,7 @@ public func normalize(path: String) -> String {
 public func splitDrive(path: String) -> (String, String) {
 #if os(Windows)
     let seq = path.map { $0 == "/" ? "\\" : $0 }
-    if seq.count > 2 && seq.starts(with: #"\\"#) && seq[2] != "\\" {
+    if seq.starts(with: #"\\"#) && seq[2] != "\\" {
```
Comment (dduan) [inline Sources/Pathos/pathAnalysis.swift:66]: this could crash if `seq` is `\\` because `seq[2]` goes out of bound.

**Comment by dduan** [comment]
Thanks for the suggestions! I think we are good to merge the first commit.

File: Sources/Pathos/pathAnalysis.swift (hunk 1 above)
Comment (Dante-Broggi) [reply Sources/Pathos/pathAnalysis.swift:66]: oops, I somehow repeatedly saw the `>` as `>=`. 🤦‍♂️

**Comment by dduan** [comment]
thank you!

**Pull request merged**